index.js

```diff
@@ -108,11 +108,16 @@ function generateCSSReferences({
 	publicPath = '',
 	attributes = {},
 }) {
+	const hasRelAttribute =
```

Comment:
Something like this would be better I think:

```js
const allAttributes = {
  ...attributes,
  rel: 'rel' in attributes ? attributes.rel : 'stylesheet'
}
```

Looks like this it's true for `rel` in this PR:

> You can now set css or js attributes to an empty string to output only the attribute name (without ="")